sql-aggregates/movie-maniacs.sql

```diff
@@ -0,0 +1,9 @@
+-- List the first and last name of all customers, plus the total amount they've spent on rentals. Order them by total paid, descending.
+
+select
+  ("c"."firstName" || ' ' || "c"."lastName") as "customer",
+  sum("p"."amount") as "totalSpent"
```

Comment:
It's generally a better plan to let the client (or web server) format full names.

Reply:
OK Done:

![image](https://github.com/jadenbertino/m2-0323-code-solutions/assets/87284530/602e3abe-e9a1-494a-99cf-2fa7a1b27a55)